optimize_workspace.sh

```diff
@@ -13,31 +13,27 @@ cargo --version
 rm -f target/wasm32-unknown-unknown/release/*.wasm
 
 # Build artifacts
-echo -n "Building artifacts in workspace..."
+echo "Building artifacts in workspace ..."
 /usr/local/bin/build_workspace.py
-echo "done."
 
-echo -n "Optimizing artifacts in workspace..."
+echo "Optimizing artifacts in workspace ..."
 mkdir -p artifacts
 TMPARTIFACTS=$(mktemp -p "$(pwd)" -d artifacts.XXXXXX)
 # Optimize artifacts
 (
   cd "$TMPARTIFACTS"
 
   for WASM in ../target/wasm32-unknown-unknown/release/*.wasm; do
-    echo -n "Optimizing $WASM..."
+    echo "Optimizing $WASM ..."
     BASE=$(basename "$WASM")
     wasm-opt -Os -o "$BASE" "$WASM"
     chmod -x "$BASE"
-    echo "done."
```

Comment:
Can we have one log line after the loop to be able to differentiate between the last optimization and whatever comes after the loop?

Reply:
Sure.